### ISSUE TITLE
Can not add new query tab /  Wrong [+] icon place on  tab list

### ISSUE THREAD

**Issue by betterapp** [HeidiSQL/HeidiSQL#1977]
**Preconditions**

* HeidiSQL version: 12.8.0.6908
* Database type and version: MySQL and MariaDB 
* OS: WIndows 11

**Describe the bug**

After adding many query tabs, query tab add button disapear. I can not find it. 
![image](https://github.com/user-attachments/assets/903ace2b-7a7c-4005-8507-d7146df06ddc)

I think better if it will looks like this.
First row for grouping access for common mysql data / functions
Second row (and additional) for query tabs only 
![image](https://github.com/user-attachments/assets/71182d46-5c51-454c-917d-bf818d622822)

**To Reproduce**

1. Add many query tabs

**Screenshots or Crash reports**

**Comment by ansgarbecker** [comment]
What you suggest in your screenshot is not doable with the underlying pagecontrol.

I suggest using the keyboard shortcut <kbd>Ctrl+T</kbd> or the menu item Query > "New query tab" for that purpose.

I know that button misplacement looks messy. I am watching out for a better (OpenSource) pagecontrol for several years now. The only good one I found is [ATTabs](https://github.com/Alexey-T/ATFlatControls). But I already tried that out and it several issues I could not resolve.

**Comment by betterapp** [comment]
is there paid solution ?

**Comment by betterapp** [comment]
What kind of component i snow used ?
Is that mean that all the tabs need to be in one row and there is no way to create two separate blocks with tabs ?

**Comment by ansgarbecker** [comment]
I'm not going to use commercial components in HeidiSQL. That would make source code publishing much more complicated.

I am using a very standard TPageControl currently, with some hacks around it to add a button and "x" buttons for closing tabs.

Note you can set the tabs to be in one row, or wrap them in multiple rows, through the context menu:
![grafik](https://github.com/user-attachments/assets/312a8d83-34eb-4969-9a56-d63f2444a7eb)

**Comment by betterapp** [comment]
Yes I know that a can have tabs in one / multi-row
But I think better is to have common function tabs in one row 
and separate row for query tabs it could make using MySQL much better

There is no possibility to create two rows and put different tabs on it ?

**Comment by ansgarbecker** [comment]
No, as said, the underlying TPageControl is not capable of that.
